gradoop-store/gradoop-hbase/src/main/java/org/gradoop/storage/impl/hbase/handler/HBaseElementHandler.java

```diff
@@ -119,14 +119,19 @@ public String readLabel(final Result res) {
    * {@inheritDoc}
    */
   @Override
-  public Properties readProperties(final Result res) throws IOException {
+  public Properties readProperties(final Result res) {
     Properties properties = Properties.create();
-    Map<byte[], byte[]> familyMap = res.getFamilyMap(CF_PROPERTIES_BYTES);
-    for (Map.Entry<byte[], byte[]> propertyColumn : familyMap.entrySet()) {
-      properties.set(
-        readPropertyKey(propertyColumn.getKey()),
-        readPropertyValue(propertyColumn.getValue()));
+    try {
```

Comment:
Make it throw, data inconsistency is much sick than declared exception. A programmer should handle those in appropriate case.